tests/test_cli.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
This file has a couple of flake8 errors

```
./tests/test_cli.py:2:5: E271 multiple spaces after keyword
./tests/test_cli.py:3:1: F401 'time' imported but unused
./tests/test_cli.py:3:1: I201 Missing newline before sections or imports.
./tests/test_cli.py:7:1: I201 Missing newline before sections or imports.
```

https://travis-ci.org/jpadilla/pyjwt/jobs/232649651